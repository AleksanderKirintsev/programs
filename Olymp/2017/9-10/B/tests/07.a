-1 -1
1 25
-1 -1
30 80
-1 -1
-1 -1
88 93
-1 -1
-1 -1
81 87
