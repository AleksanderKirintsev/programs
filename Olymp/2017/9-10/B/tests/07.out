-1 -1
1 25
-1 -1
30 80
-1 -1
-1 -1
81 93
-1 -1
-1 -1
81 87
